composer.json

```diff
@@ -3,11 +3,15 @@
     "description": "Laravel plugin that helps authenticate with the auth0 service",
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0",
```

Comment:
I just had to bump this to prevent confusion. Because installing with PHP 5.4 will not work currently.

```
Problem 1
    - auth0/auth0-php 5.6.0 requires php ^5.5 || ^7.0 -> your PHP version (7.3.8) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - auth0/auth0-php 5.6.0 requires php ^5.5 || ^7.0 -> your PHP version (7.3.8) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - Installation request for auth0/auth0-php ^5.6.0 -> satisfiable by auth0/auth0-php[5.6.0].
```